src/linker/cli.py

```diff
@@ -24,13 +23,19 @@ def linker():
 
 
 @linker.command()
-@click.argument(
-    "pipeline_specification",
+@click.option(
```

Comment:
I snuck these in after confirming with Zeb that "required options" are better than arguments (because they are non-ordered and also they can have a help string"

Reply:
Implemented in followup PR